clang-tools-extra/clang-tidy/readability/RedundantInlineSpecifierCheck.cpp

```diff
@@ -47,6 +47,9 @@ AST_POLYMORPHIC_MATCHER_P(isInternalLinkage,
     return VD->isInAnonymousNamespace();
   llvm_unreachable("Not a valid polymorphic type");
 }
+
+AST_MATCHER(clang::VarDecl, hasInitialization) { return Node.hasInit(); }
```

Comment:
why not use `hasInitializer(expr())` instead of adding next matcher.

Reply:
Simply didn't know this exists. I'll update my code with your suggestion. Thank you